### ISSUE TITLE
Exhibition filter

### ISSUE THREAD

**Issue by edgartdata** [ycba-cia/blacklight-collections2#192]
Check that exhibitions in planning process are not coming through LIDO & BlackLight. The BlackLight group consensus was that ideally LIDO & BlackLight would not publish exhibitions that have closed to the public less than a year from today.

Today, ‘Art in Focus: Women at Yale’ (April-August 2020 at the YCBA) exhibitionID 1024 appears in objectID 1336 exhibition history in BlackLight. 

It might be because its TMS exhibition record did not have a value in the Display Date field. @edgartdata added the display date and will check if that prevents this exhibition to come through LIDO and BlackLight.

**Comment by edgartdata** [comment]
I have verified that the lack of data in the Display Date field is not the reason why  ‘Art in Focus: Women at Yale’ (April-August 2020 at the YCBA) exhibitionID 1024 still appears in BlackLight today.

**Comment by yulgit1** [comment]
Trying to understand, you're saying you added the display date, let coboat update, and the exhibit is still in in the LIDO?  As it is here updated 5/7 (yesterday)?

https://harvester-bl.britishart.yale.edu/oaicatmuseum/OAIHandler?verb=GetRecord&identifier=oai:tms.ycba.yale.edu:61974&metadataPrefix=lido

**Comment by edgartdata** [comment]
@yulgit1 yes, correct. I actually added the display date to TMS several days ago and it is still in LIDO. Should we still proceed with fixing this in COBOAT?

**Comment by yulgit1** [comment]
Yes, seems like COBOAT is the best place to suppress these.

**Comment by edgartdata** [comment]
@EdwardTown1 @flapka Can you recommend an appropriate cut off date for publishing exhibition citations (in exhibition history)?  

Currently David's COBOAT script only takes into consideration the year the exhibition opens in. In other words, exhibitions that open in 2020, including the ones that will open between today and 2021, are published to BlackLight. This scenario is not the best because we know that exhibitions that will open between now and the end of the year are most likely to have their TMS records in a state of flux until the exhibitions open.

On the other hand, if we keep with years as our only selection criteria and select 2019 as our cut off date, then we would not publish the most recent exhibitions organized at the Center/external exhibitions in which YCBA objects have been included.

I would think that taking into consideration month & year would be a more appropriate way to go. On the flip side, that is probably more complex to put in place in COBOAT.

Questions to consider:

- Do we want to filter on month/year? or just year?

- If we choose month/year, is it for the opening date? For the closing date?

- Do we want to publish the most recent exhibitions? Even if they are not closed yet?

- If we don't want to include the exhibitions that are currently open, what would be our minimum buffer? One month after the exhibitions close? Two months? More?

**Comment by flapka** [comment]
@edgartdata @EdwardTown1 

I think I missed the original discussion on this issue. Regardless, I would prefer for Blacklight's exhibition information to include everything through exhibitions currently on display. 

RB hasn't added data to MARC for exhibitions in the future, so that issue is currently moot for us.

**Comment by EdwardTown1** [comment]
@edgartdata @flapka Very happy with Francis's suggestion, my only thought
is what about objects that were supposed to travel this year (or next!) to
shows that are either postponed or cancelled altogether.  The latter is
true for: http://10.5.96.187:3000/catalog/tms:12454

'Renaissance Masters' is cancelled. Which is a great shame.

E T

**Comment by edgartdata** [comment]
My suggestion would be to invite Nancy and Corey to our meeting Friday and discuss a process whereby TMS reflects postponed or cancelled exhibitions.

**Comment by edgartdata** [comment]
Given our conversation with Nancy and Corey on May 15, in order to publish to BlackLight all the exhibitions up to the ones currently on display except the ones that are cancelled, the group decided to make use of the TMS exhibition status flag 'cancelled'. The TMS exhibition record will stay as is with its dates of exhibition (for registrarial reasons) so David Parsell will need to add the new exhibition status flag 'cancelled' to his filtering mechanism.

The _Renaissance _Master__ exhibition has been cancelled and should disappear from Devereux's miniature portrait once David puts in place the additional filter: http://10.5.96.187:3000/catalog/tms:11628

**Comment by edgartdata** [comment]
After further conversation with the registrars, it was agreed that only exhibitions with the status of 'Completed' would be published to BlackLight.
This is contingent on David Parsell programmatically giving this status to the 682 exhibitions that are ready for him in the TMS object package _CIA-exhibitions with venue end date of or before 12/31/2017 (682)_

**Comment by edgartdata** [comment]
Updates: 

- David is done programmatically giving the status of 'Completed' to the 682 exhibitions in TMS.

- Nancy MacGregor was able to review all the exhibitions post-12.31.2017 and assign the status of 'Completed' whenever appropriate. So today, we have 727 exhibitions ready for publication to BL, from 1965 (_English Drawings and Watercolors from the collection of Mr. and Mrs. Paul Mellon_) to December 2020 (_Lynette Yiadom-Boakye_). 

- Also today, David Parsell installed the ‘Completed’ code for the exhibitions in blacklight AWS and bac6-dev and we should see this data coming through BL tomorrow.

@EdwardTown1 @edgartdata to check on that tomorrow.
